src/common/UserDefaults.h

```diff
@@ -163,8 +163,10 @@ enum DefaultKey
     DontShowAudioErrorsAgain,
 
     // OSC (Open Sound Control)
-    StartOSC,
-    OSCPort,
+    StartOSCIn,
+    StartOSCOut,
+    OSCPortIn,
+    OSCPortOut,
```

Comment:
this is absolutely fine but if anyone used the old keys they may get an error with the new keys. Nightlies gonna nightly so no change required but if that happens the fix is reset a param or manually edit the xml to restore.